docs/backend/behaviors.md

```diff
@@ -11,12 +11,340 @@ myst:
+  />
+  ```
+
+After a restart of Plone, the behavior can be added to the content type in the {guilabel}`Content Types` control panel.
+The add and edit forms contain a new field `Price (net)`.
+
+This field is not displayed in most views.
+To display the data entered in this field, you need to modify the page template by adding the `price_net` field as `context.price_net`.
+To access the `price_vat` and `price_gross` fields from a browser view, you need to get the adapter from the context of the view:
+
+```python
+from .price import IPriceBehavior
+
+class SomeViewClass:
+
+    def vat(self):
+        adapter = IPriceBehavior(context)
+        return adapter.price_vat
+
+    def gross(self):
+        adapter = IPriceBehavior(context)
+        return adapter.price_gross
+```
+
+### Creating a behavior with PloneCLI
+
+To add a behavior to your add-on, you can use PloneCLI as follows:
+
+```shell
+plonecli add behavior
+```
+
+This will create the behavior Python file in the `behaviors` folder, where you can define your behaviors schema fields, and registers the behavior in the `configure.zcml`.
+
+### Further reading on working with behaviors
 
 ```{seealso}
 See the chapter {ref}`training:behaviors1-label` from the Mastering Plone 6 Training.
 ```
 
-```{todo}
-Contribute to this documentation!
-See issue [Backend > Behaviors needs content](https://github.com/plone/documentation/issues/1302).
-```
\ No newline at end of file
+## How behaviors work
+
+```{note}
+Skip this section if you do not want to dive deeper into the internals of behaviors.
+You do not *need* to know this, but it may help if you run into problems.
+```
+
+In Plone, behaviors can be globally enabled on content types at runtime.
+With add-ons, behaviors can even be enabled even on a single content object or for a whole subtree in the content hierarchy.
+
+### Interfaces and adapters
+To explain interfaces and adapters, let's begin with an analogy using electrical systems.
+
+An electrical outlet provides an interface through which electricity passes.
+When you travel to another country, you may need an outlet adapter for the outlet (the interface).
+For example, assume you have a device that has plug for Schuko outlets, and in Italy there are Type L outlets.
+If we were to represent the behavior of choosing the correct power adapter in Plone, you would do the following.
+
+-   Call `getAdapter(context, ITypeL)` to determine the type of adapter needed for the electric outlet's interface.
+-   Call `getAdapter(context, ISchuko) to determine the type of adapter that will accept your device's plug.
+-   You choose the correct electrical adapter to use, and plug that into the wall outlet.
+    This is similar to calling an abstract factory which returns the correct adapter to use.
+    This interface provides an ISchuko adapter.
+-   Finally to use the new interface, you plug in your Schuko plug into the adapter to get power to your device.
```

Comment:
```suggestion
-  You need an electrical adapter for your Schuko plug.
  1. you look at the outlet and see it is TypeL.
  2. you look in your box with different adapters and choose one the correct electrical adapter to use, and
  3. plug that into the wall outlet.
  4. Finally you can use your Schuko providing device on an Italian TypeL outlet.
-  In Python you call `getAdapter(context, ISchuko)` (context is here the outlet) which then 
   1. determine the type of interface provided by the `context` and as a result it finds `ITypeL`.
   2. looks in the component registry if there is a class that adapts to `ITypeL` and at the same time provides the requested `ISchuko`.
   3. initializes the adapter class with the context and return it as the result
   4. Finally the `ISchuko` providing adapter can be used on a `ITypeL` providing context.

This process of choosing the right adapter based on the information of the context and the requested interface implements the design pattern of an abstract factory.
```

Reply:
Careful of indentation of nested lists. Make sure they align with the first letter of the parent list item. This is one reason I always indent list items 4 spaces.

Also you might need to add a blank line between a nested list. In reST this was strictly enforced, but in the last year or so and with MyST, I am not sure. I just do it by habit now. Check the Netlify build to make sure it looks OK.

```
- Parent List Item

    #.  Don't use actual numbers because they often need renumbering in the future.
    #.  Instead use `#.`.
    #.  Plus when you hit 10 or more enumerated items, you still indent the right number of spaces.

- Parent List Item
```

Reply:
Thank you for the revision. This is a lot better.